src/main/java/mate/academy/dao/impl/MovieSessionDaoImpl.java

```diff
@@ -0,0 +1,51 @@
+package mate.academy.dao.impl;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.util.List;
+import java.util.Optional;
+import mate.academy.dao.MovieSessionDao;
+import mate.academy.exception.DataProcessingException;
+import mate.academy.lib.Dao;
+import mate.academy.model.MovieSession;
+import mate.academy.util.HibernateUtil;
+import org.hibernate.Session;
+import org.hibernate.query.Query;
+
+@Dao
+public class MovieSessionDaoImpl extends AbstractDao implements MovieSessionDao {
+    public MovieSessionDaoImpl() {
+        super(HibernateUtil.getSessionFactory());
+    }
```

Comment:
Don't create a default constructor when it is not needed. Since you are not doing any additional operations other than calling the superclass constructor, this default constructor is redundant and can be removed.